### DISCUSSION TITLE
Τεκμηρίωση εγκατάστασης του λειτουργικού συστήματος

### DISCUSSION THREAD

**Discussion by AlexandrosV20** [courses-ionio/hci#1622] [Announcements]
Λέτε ότι για την βαθμολόγηση του λετουργικού πρέπει να γίνει το εξής : Για την βαθμολόγηση του λειτουργικού συστήματος θα πρέπει να γίνει ένα asciinema με boot log, system info (π.χ, journalctl -b, neofetch). Τι εννοείτε ακριβώς πως θα γινει αυτό. Αν πατήσω  το journalctl -b που λετε βγάζει μια τεράστια λίστα. Αυτή την λίστα θέλετε να την κάνουμε ρέκορντ ;

**Comment by epidrome** [comment]
η εντολή έχει φίλτρα για τον έλεγχο της εξόδου της https://man.archlinux.org/man/journalctl.1

αν η έξοδος είναι εκτενής μπορεί να γίνει είσοδος σε μια εντολή ανάγνωσης όπως η less: https://man.archlinux.org/man/less.1
